TSC-Charter.md

```diff
@@ -47,8 +47,8 @@ TSC by a standard TSC motion and vote. A TSC member may be removed from the
 TSC by voluntary resignation, by a standard TSC motion, or in accordance to the
 participation rules described below.
 
-Changes to TSC membership should be posted in the agenda, and may be suggested
-as any other agenda item.
+Changes to TSC membership should be posted in the agenda, and will be then discussed
+at the next TSC meeting when all members are able to attend.
```

Comment:
This is a functional change to how the TSC works. This should probably go through the process outlined for modifying this document.

Reply:
It may be, and if so this point should be discussed and agreed upon by the TSC.  In other governance documents I have reviewed, it is common to ensure some percentage (sometimes all members) of the TSC are present when discussing adding new members.  Following this model may (or may not) be something the TSC would like to consider.